fingerprintauthhelper/src/main/kotlin/com/pro100svitlo/fingerprintAuthHelper/FahManager.kt

```diff
@@ -303,11 +290,11 @@ internal class FahManager(@NonNull c: Context, l: FahListener?, keyName: String,
             }
 
             try {
-                mKeyStore!!.load(null)
+                keyStore.load(null)
```

Comment:
and here you are "loading" to null local variable instead property variable. or I am missing the idea of this?